packages/concerto-core/test/data/parser/classdeclaration.badidentifierprefix.cto

```diff
@@ -11,16 +11,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
-/*eslint-disable no-unused-vars*/
```

Comment:
That's a weird change. Git hiccup? Maybe it's just a wrongly identified file rename.

Reply:
Yes, this is weird. This wasn't a rename, I deleted `functiondeclaration.throwsexception.js` and created a new file `classdeclataion.badidentifierprefix.cto`, they just happen to have lines in common.